### PR TITLE
Add recipes for mleap and nose-exclude

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
Required for https://github.com/conda-forge/staged-recipes/pull/8072

cc @jaroslawk

**Comment by xhochy** [comment]
@conda-forge/staged-recipes Build is green, please give this a review.

File: recipes/nose-exclude/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - setuptools
+
+test:
+  imports:
+    - nose_exclude
+
+about:
+  home: https://github.com/kgrandis/nose-exclude
+  license: GNU Library or Lesser General Public (LGPL)
```
Comment (chrisburr) [inline recipes/nose-exclude/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: LGPL-2.1
```

File: recipes/nose-exclude/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: LGPL
```
Comment (chrisburr) [inline recipes/nose-exclude/meta.yaml]:
```suggestion
  license_family: GPL
```

File: recipes/nose-exclude/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: f78fa8b41eeb815f0486414f710f1eea0949e346cfb11d59ba6295ed69e84304
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - nose
```
Comment (chrisburr) [inline recipes/nose-exclude/meta.yaml]: I think `nose` is only a runtime dependency:
```suggestion
```

File: recipes/mleap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    - mleap.sklearn.decomposition
+    - mleap.sklearn.ensemble
+    - mleap.sklearn.extensions
+    - mleap.sklearn.feature_extraction
+    - mleap.sklearn.preprocessing
+    - mleap.sklearn.tree
+
+about:
+  home: https://github.com/combust/mleap/
+  license: Apache Software
```
Comment (chrisburr) [inline recipes/mleap/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/nose-exclude/meta.yaml (hunk 2 above)
Comment (xhochy) [reply recipes/nose-exclude/meta.yaml]: Looks like a problem with `conda skeleton pypi`. I'll file an upstream issue.

File: recipes/mleap/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb384a239747f319297a0b4f2bb5c076f01cfd942d7f72a173494606f402fdad
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/mleap/meta.yaml]: I think this can be `noarch`
```suggestion
  number: 0
  noarch: python
```

File: recipes/mleap/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - nose-exclude >=0.5.0
```
Comment (chrisburr) [inline recipes/mleap/meta.yaml]: Are all of these needed as host requirements? You probably only need `python` and `pip`.

Comment (xhochy) [reply recipes/mleap/meta.yaml]: No, they aren't, sorry. Fixed this.

**Comment by xhochy** [comment]
@chrisburr Fixed all your comments and the build is green again.

**Pull request merged**